lib/event_flow/ui/event_flow_section.dart

```diff
@@ -249,8 +279,23 @@ class _EventFlowSessionPoint extends StatelessWidget {
         width: radius,
         child: DecoratedBox(
           decoration: BoxDecoration(shape: BoxShape.circle, color: pointColor),
+          child: Padding(
+            padding: const EdgeInsets.all(2),
```

Comment:
Values ​​must be multiples of 4.
```suggestion
            padding: const EdgeInsets.all(4),
```

Reply:
I agree with you about it. But when I make It 4, it looks too small the understand the flag.

### 2px Padding

![Screen Shot 2022-03-11 at 19 38 44](https://user-images.githubusercontent.com/26827403/157909510-67183aa6-480a-4fa1-bc6e-ae6ddb9a2f17.png)
![Screen Shot 2022-03-11 at 19 39 32](https://user-images.githubusercontent.com/26827403/157909636-61090746-11e4-4a77-a79f-2032a7d3dfaa.png)


-----

### 4px Padding

|![Screen Shot 2022-03-11 at 19 36 10](https://user-images.githubusercontent.com/26827403/157909276-917ff743-2578-4a39-ba69-ccc439484731.png)
![Screen Shot 2022-03-11 at 19 35 52](https://user-images.githubusercontent.com/26827403/157909317-5d7ab398-5f6d-4207-8ce3-5b8ec829f7f9.png)